src/main/java/wooteco/subway/controller/LineController.java

```diff
@@ -0,0 +1,50 @@
+package wooteco.subway.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import wooteco.subway.dto.LineRequest;
+import wooteco.subway.dto.LineResponse;
+import wooteco.subway.service.LineService;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+public class LineController {
+
+    private final LineService lineService;
+
+    public LineController(LineService lineService) {
+        this.lineService = lineService;
+    }
+
+    @PostMapping("/lines")
+    public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
+        LineResponse lineResponse = lineService.create(lineRequest.getName(), lineRequest.getColor());
+        return ResponseEntity.created(URI.create("/lines/" + lineResponse.getId())).body(lineResponse);
+    }
+
+    @GetMapping("/lines")
+    public ResponseEntity<List<LineResponse>> showLines() {
+        List<LineResponse> lineResponses = lineService.findAll();
+        return ResponseEntity.ok().body(lineResponses);
```

Comment:
```suggestion
        return ResponseEntity.ok(lineResponses);
```

사소한 것이긴 하지만, OK 응답의 경우 이렇게 shortcut을 지원하고 있어요. 또는, `ResponseEntity`로 감싸지 않고 응답 객체를 그대로 리턴해도 됩니다.

Reply:
지난 체스 미션을 스프링 프레임워크로 적용하면서 해당 내용을 분명 학습했는데..코드를 작성하다보니 다시 `body()`에 데이터를 넣어줬었네요😢

해당 내용도 수정했습니다!!